Library/Formula/kubernetes.rb

```diff
@@ -0,0 +1,36 @@
+class Kubernetes < Formula
+  homepage "http://kubernetes.io/"
+  url "https://github.com/GoogleCloudPlatform/kubernetes/archive/v0.9.3.tar.gz"
+  sha256 "b3a6008f1db7f9153c6396cc1c3e890ca59450392556c3f823226738ab6a91fc"
+
+  bottle do
+    sha1 "TODO" => :yosemite
+    sha1 "TODO" => :mavericks
+    sha1 "3c5f7480d2f790df59800a771ff1ea2efae8978b" => :mountain_lion
+  end
+
+  depends_on "go" => :build
+
+  option "with-verbose", "Be verbose while building binaries"
```

Comment:
I can remove the option, that's not a problem, but default **on** or **off** then?